scaffold/test.template.js

```diff
@@ -0,0 +1,22 @@
+/* globals describe it */
+const should = require('should');
+
+const zapier = require('zapier-platform-core');
+
+// Use this to make test calls into your app:
+const App = require('../../index');
+const appTester = zapier.createAppTester(App);
+zapier.tools.env.inject();
+
+describe('My App', () => {
+  it('should run <%= TYPE %>s.<%= KEY %>', done => {
+    const bundle = { inputData: {} };
+
+    appTester(App.<%= TYPE %>s.<%= KEY %>.<%= MAYBE_RESOURCE %>operation.perform, bundle)
+      .then(results => {
+        should.exist(results)
```

Comment:
Missed a semicolon at the end of line.

Reply:
good eye! Since these don't parse as valid JS, i'm flying a little blind without my tools, haha.